Restore procedure docs description (#3300)
scylladb/scylla-manager: docs/source/restore/restore-tables.rst

==============
Restore tables
==============

> In order to restore tables' contents (rows) use :ref:`sctool restore <sctool-restore>` with ``--restore-tables`` flag.

Reviewer: ```suggestion
To restore the content of the tables (rows), use the :ref:`sctool restore <sctool-restore>` command with the ``--restore-tables`` flag.
```

Prerequisites
=============

> #. Scylla Manager with CQL credentials to restore destination cluster.

Reviewer: ```suggestion
* Scylla Manager with CQL credentials to restore destination cluster.
```

> #. Restoring tables' contents assumes that correct schema is already present in destination cluster.

Reviewer: ```suggestion
#. Restoring the content of the tables assumes that the correct schema is already present in the destination cluster.
```

   Scylla Manager does NOT validate that, so it's user responsibility to ensure it. The only form of validation
   that Scylla Manager performs is checking whether restored tables are present in destination cluster,
   but it does not validate their columns types nor other properties. In case destination cluster is missing correct schema,
   it should be :doc:`restored <restore-schema>` first.

> #. It is strongly advised to restore tables' contents only into `truncated <https://docs.scylladb.com/stable/cql/ddl.html#truncate-statement>`_ tables.

Reviewer: ```suggestion
* It is strongly advised to restore the contents of the tablesonly into `truncated <https://docs.scylladb.com/stable/cql/ddl.html#truncate-statement>`_ tables.
```

   Otherwise, restored tables' contents might be overwritten by the already existing ones.
   Note that an empty table is not necessarily truncated!

Follow-up action
================

> After successful restore it is important to perform necessary follow-up action. In case of restoring tables' contents,

Reviewer: ```suggestion
After the successful restore, it is important to perform the necessary follow-up action. After restoring the content of the tables,
```

> user should :ref:`repair <sctool-repair>` all restored keyspaces and tables.

Reviewer: ```suggestion
you should :ref:`repair <sctool-repair>` all restored keyspaces and tables.
```

> Without it restored tables' contents might not be visible and querying it can return various errors.

Reviewer: ```suggestion
Without the repair, the restored content of the tables might not be visible, and querying it can return various errors.
```

> Without the repair, the restored content of the tables might not be visible, and querying it can return various errors.

Reviewer 2: To be confirmed:
- do we require end-user to bring back `gc_grace_seconds` after the repair ?
- or we include that into the automated restore
